### ISSUE TITLE
[Request] Add support for Tiger Lake / Alder Lake / Raptor Lake 

### ISSUE THREAD

**Issue by nocontent06** [NootInc/NootedBlue#6]
PLEASE! Add support for:

i3/5/7/9 11xxx
i3/5/7/9 12xxx
i3/5/7/9 13xxx

iGPUs

Also CPUs like i5-1240P, i7-1260P, ...

I'd really appreciate it 🙏

**Comment by Zormeister** [comment]
Support for these generations is purely hypothetical, but, plausible, and may be worked on at a later date as our main focus are integrated graphics that has a driver, example: Braswell (Generation 8) on the AppleIntelBDWGraphics stack

**Comment by nocontent06** [comment]
Yeah I know. I am aware of there Iris Xe GPUs. Because many people these days have a Laptop with Intel 11th Gen and up. And since intel will continue using Iris Xe, it would be a *huge* achievement, in terms of bringing unsupported GPUs to macOS.

**Comment by macsupermac** [comment]
I hope that Add support for Tiger Lake or Later,Intel Iris Xe Graphcis is powerful.It's a great pity that Apple give up supporting 11th or Later.Thanks!!!

**Comment by theandroidguy99** [comment]
Would love to see the 12th gen mobile chips getting support for macos as 11th gen chips already have support, I used to have hackintosh with a i5 7th gen laptop but now with a i3 12th gen machine, I really miss macOS

**Comment by nocontent06** [comment]
Yo guys! I follow your posts on insanelymac.com and i really appreciate your work!
Thanks to y'all :)

**Comment by ChitoKim** [comment]
Things have changed a lot thanks to AppleIntelTGL kext leaks.

**Comment by Overc1ocker** [comment]
Stupid question,

But you guys said you are willing to work on ` integrated graphics that has a driver` so I have a question and I would love to potentially help develop the answer.

Can Haswell graphics be hacked to run on Broadwell drivers? This would specifically benefit 2 machines that are a nightmare to get running on newer OSes because of 2 different metal compiler versions being used (3802 and 31001) for the integrated and discrete gpu. Running broadwell drivers with haswell would fix this issue.

**Comment by Zormeister** [comment]
haswell (Gen 7.5) and broadwell (Gen 8) architectures have similar pieces here and there, read the [i915](https://github.com/torvalds/linux/tree/master/drivers/gpu/drm/i915) and [Mesa](https://gitlab.freedesktop.org/mesa/mesa/-/tree/main/src/intel?ref_type=heads) codebase if you're curious

for the love of god PLEASE don't use this repo to do any work like that, i'm going to be archiving the repo once i finish looking for any remnants of unfinished code
